tests/test_advanced_options.py

```diff
@@ -34,7 +34,7 @@ def test_skip_report(analysis_data):
 # Polarion TC 374
 def test_custom_rules(analysis_data):
     application_data = analysis_data['jee_example_app']
-    custom_rule_path = os.path.join(os.getenv(constants.PROJECT_PATH), 'data/xml', 'weblogic-custom.windup.yaml')
```

Comment:
Is the `weblogic-custom.windup.yaml` rule being used anywhere else?

If not it could be removed

Reply:
Removed